webapp/niapp.yaml

```diff
@@ -235,7 +255,32 @@ paths:
           $ref: '#/responses/Unauthorized'
         404:
           $ref: '#/responses/NotFound'
+        default:
+          $ref: '#/responses/Error'
 
+  '/webapps/duplicate/{sourceId}':
```

Comment:
At first I was thinking this should be a `POST /duplicate-webapps` to match our HTTP conventions, but then I realized sourceId is a path parameter and that there's already a `POST /webapps/{id}/duplicate/{sourceId}`. Would it makes sense to merge those two routes into a `POST /webapps/{id}/duplicate` where source ID is in the path and destination ID is optional in the body?

That way, the URL is implying the "duplicate" action on the `/webapps/{id}` resource.

Reply:
We could do that. I'm not sure what it would take to drop support for the existing route, though. We would probably need to keep it for at least a little while. Do you have an issue keeping both routes for now?

Reply:
No, that sounds fine to me.

Reply:
After an offline conversation with @spanglerco, we decided there wasn't a pressing reason to try to have the new duplicate route be dual-purpose to handle specifying an ID to duplicate only the content or specifying a name + workspace to duplicate content + metadata. This simplifies the documentation, testing, and implementation.

I will, however, change this new route to be `/webapps/{sourceId}/duplicate`.